runner.go

```diff
@@ -143,13 +143,26 @@ func PassThrough(types ...Type) Runner {
 	if len(types) == 0 {
 		return passThroughSingleton
 	}
-	return &passThrough{types}
+	return &passThrough{types, StringsSet{}}
```

Comment:
no need to initialize scopes :)
```suggestion
	return &passThrough{ReturnTypes: types}
```